repos/system_upgrade/common/actors/cloud/checkhybridimage/libraries/checkhybridimage.py

```diff
@@ -39,27 +45,40 @@ def is_azure_agent_installed():
     return has_package(InstalledRPM, agent_pkg)
 
 
+def has_efi_partition():
+    """
+    Check whether ESP partition exists and is mounted.
+    """
+
+    return os.path.exists(EFI_MOUNTPOINT) and os.path.ismount(EFI_MOUNTPOINT)
```

Comment:
@dkubek checking the actor, it does not produce any report any depends on scanners. why not to move this actor then to the FactsPhase instead of keeping it in ChecksPhase?

Reply:
It would be more complicated, but this info should already be in the StorageInfo message.

Reply:
Moved to scan  phase